build.gradle

```diff
@@ -24,7 +24,8 @@ plugins {
     id "com.github.spotbugs" version "4.7.1"
     id "io.codearte.nexus-staging" version "0.30.0"
```

Comment:
We can get rid of this too right?

Reply:
Yea, I left it in so we could fall back to it if necessary